hgtv/views/index.py

```diff
@@ -18,9 +19,27 @@ def longdate(date):
     return utc.localize(date).astimezone(app.config['tz']).strftime('%e %B %Y')
 
 
+def jsonify_featured_channel(data):
+    channels_dict = []
```

Comment:
I suggest `channel_dicts`, since `channels_dict` suggests it's a `dict`.

Reply:
Fixed